conf/core/ConfigOptions.asciidoc

```diff
@@ -1001,8 +1001,10 @@ See also:
 
 * Data Type: list
 * Default Value:
-** `hoot::RemoveMissingElementsVisitor`
+** `hoot::MultilineStringMergeRelationCollapser`
+** `hoot::RemoveDuplicateWayNodesVisitor`
 ** `hoot::InvalidWayRemover`
+** `hoot::RemoveMissingElementsVisitor`
 ** `hoot::PoiPolygonInvalidReviewNodeRemover`
```

Comment:
Moved `MultilineStringMergeRelationCollapser` up to the top of the post conflate chain in order to get the test output I wanted. This was the guy added in to help remove he ms relations from the cumulative conflate work. I think it makes more sense there anyway. I did have to put `RemoveDuplicateWayNodesVisitor` and `InvalidWayRemover` immediately after it to keep things sane...don't completely understand why yet, though.